### PR TITLE
Remove rpy2, fix actogram plotting and test

### PR DESCRIPTION
**Pull request by justinshenk**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "doc_module": ("traja",),
     "reference_url": {
         "numpy": "http://docs.scipy.org/doc/numpy",
-        "scipy": "http://docs.scipy.org/doc/scipy/reference",
+        # "scipy": "http://docs.scipy.org/doc/scipy/reference",
```
Comment (WolfByttner) [inline docs/source/conf.py:71]: Are we meant to remove the scipy reference?

File: traja/plotting.py [hunk 2]
```diff
@@ -984,10 +990,12 @@ def plot_actogram(
        Curr Protoc Mouse Biol. 2015;5(3):271-281. Published 2015 Sep 1. doi:10.1002/9780470942390.mo140229
 
     """
-    after_plot_args, _ = _get_after_plot_args(**kwargs)
+    assert isinstance(series, pd.Series)
```
Comment (WolfByttner) [inline traja/plotting.py:993]: Why are we removing the assert?

Comment (justinshenk) [reply traja/plotting.py:993]: we moved the assert a few lines up

File: docs/source/conf.py (hunk 1 above)
Comment (justinshenk) [reply docs/source/conf.py]: Fixed in e456902.

**Pull request merged**